docs/widgetbook-cloud/quick-start.mdx

```diff
@@ -25,10 +25,11 @@ This quick start guide will help you set up and use Widgetbook Cloud efficiently
 2. Install and configure the Widgetbook CLI.
 3. Ensure that your application's resources are accessible by configuring CORS settings for your backend services.
```

Comment:
Step 3, Point 3: Navigate to the project's details page might be misleading. 

1. User has to go to the project
2. User has to go to _Settings_
3. User can fetch API key

Reply:
agreed. fixed